docs/src/main/sphinx/connector/hive.md

```diff
@@ -1251,11 +1251,11 @@ session property `<hive-catalog>.dynamic_filtering_wait_timeout`.
 ```{include} table-redirection.fragment
 ```
 
-The connector supports redirection from Hive tables to Iceberg
-and Delta Lake tables with the following catalog configuration properties:
+The connector supports redirection from Hive tables to Iceberg, Delta Lake and Hudi tables with the following catalog configuration properties:
```

Comment:
Can we wrap the lines so that it within 90 characters in a line

Reply:
I think it should be 80, not 90.